<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>637</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>JetBrains Mono</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>BandMap</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>358</width>
         <height>527</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="font">
           <font>
            <family>JetBrains Mono</family>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoominButton">
        <property name="text">
         <string>In</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomoutButton">
        <property name="text">
         <string>Out</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="clear_spot_olderLabel">
        <property name="text">
         <string>Clear older than</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="clear_spot_olderSpinBox">
        <property name="specialValueText">
         <string>Never</string>
        </property>
        <property name="suffix">
         <string> min(s)</string>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
